charts/nautobot/values.yaml

```diff
@@ -516,6 +516,38 @@ nautobots:
 #     - "/graphql"
 #   initNautobot: false
 
+# Default values for Network Policy
+# @default -- See values.yaml
+networkPolicy:
+  # -- [[ref](https://kubernetes.io/docs/concepts/services-networking/network-policies/)]Enables a Network Policy for Nautobot pods
+  enabled: false
+  # -- Permit all Nautobot pods to access any destination in any port
+  allowAllEgress: false
+  # -- Allow outside connectivity to http/https ports of Nautobot Web
+  allowExternalIngress: true
```

Comment:
Perhaps `allowAllIngress`? I was a bit confused about what this variable does.

Reply:
It is not all ingress though! Pods from other namespaces don't get access. It only allows access from IP addresses not part of the kubernetes cluster. 
tbh, I changed from `allowAllIngress` to `allowExternalIngress` in order to be more understandable of the different scenarios :)